cli/azd/cmd/templates.go

```diff
@@ -114,15 +115,15 @@ func (tl *templateListAction) Run(ctx context.Context) (*actions.ActionResult, e
 		columns := []output.Column{
 			{
 				Heading:       "Name",
-				ValueTemplate: "{{.Name}}",
+				ValueTemplate: `{{if ne .Name ""}}{{.Name}}{{else}}{{.Title}}{{end}}`,
 			},
 			{
 				Heading:       "Source",
 				ValueTemplate: "{{.Source}}",
 			},
 			{
 				Heading:       "Repository Path",
-				ValueTemplate: "{{.RepositoryPath}}",
+				ValueTemplate: `{{if ne .RepositoryPath ""}}{{.RepositoryPath}}{{else}}{{.RepoSource}}{{end}}`,
 				Transformer:   templates.Hyperlink,
 			},
```

Comment:
We already have a schema for what a template source looks like - we should enforce this instead of adding new optional properties like name vs title, etc.